### ISSUE TITLE
Adding an object via objects/add ignores tags

### ISSUE THREAD

**Issue by Rafiot** [MISP/MISP#3124]
How to reproduce:

POST - https://mispbeta.circl.lu/objects/add/9830/55

```
{
  "Attribute": [                                           
    {                        
      "Tag": [               
        {                                                  
          "name": "fail2ban"                               
        }                                                  
      ],                     
      "category": "Network activity",                      
      "disable_correlation": false,                        
      "object_relation": "asn",                            
      "to_ids": false,       
      "type": "AS",                                        
      "value": "1234"        
    }                        
  ],                                                       
  "description": "Autonomous system object describing an autonomous system which can include one or more network opera
tors management an entity (e.g. ISP) along with their routing policy, routing prefixes or alike.",                    
  "distribution": 5,         
  "meta-category": "network",
  "name": "asn",             
  "sharing_group_id": 0,     
  "template_uuid": "4ec55cc6-9e49-4c64-b794-03c25c1a6587", 
  "template_version": 4,     
  "uuid": "c11245b5-d021-4eb6-8a9c-a973327e415b"           
}
```                                                   

Properly creates the object, but no tag is set

**Comment by iglocska** [comment]
That is correct, tags were never in the scope of the objects/add API. I can change it, but do we really want it? Except for the /events/add and /events/edit, all of the add/edit endpoints are atomic.

It's a bit of a slippery slope - do we do the same with proposals? What about sightings? Object references?

**Comment by Rafiot** [comment]
For the end user, it is confusing that you can add an event containing everything, but that is is not the case for other "types".

I can hide it in pymisp and do two or more queries depending on what is in the json blob.

For references, I see the point, as it is a reference to an other (potentially non existing) object. But Tags are seen as part of an attribute/event(/object in the future)

**Comment by iglocska** [comment]
In general this should be the case for everything. We allow full events via
the events api because of the sync mechanism using it. In general you
should always add atomic objects. I can make an exception for attribute
tags, but the question remains: What about sightings and proposals? Do we
also add tags to /attributes/add? What about adding users directly via
/organisations/add? There be dragons.

**Comment by Rafiot** [comment]
I'll just add a hook in PyMISP to automatically catch the sub-objects. Thinking of it, it is a totally valid, easy, and clean solution to support that functionality.

On the other end, on MISP side itself, it might be good to inform the used passing the whole object via the API that a part of it is silently dropped.

**Comment by ghost** [comment]
Miss this feature too and worked around with fetching the existing event, add my object including tags and do an update_event() on this.

Regarding #3314 - I would also use attribute specific tagging to control the data processing like SIEM Alerts, Blocklist generation and so on. Regardless of dealing with an attribute itself or and object-attribute. 

For this use case it would be very nice to have API based support for add/remove tags from attributes / object-attributes.

**Comment by mokaddem** [comment]
This feature is implemented with the merge of #6143.
